ios/Plugin/Plugin.swift

```diff
@@ -126,6 +126,52 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
         }
     }
 
+    @objc func getBeaconsPermissionStatus(_ call: CAPPluginCall) {
+        getLocationPermissionsStatus(call)
+    }
+
+    @objc func requestBeaconPermissions(_ call: CAPPluginCall) {
+        DispatchQueue.main.async {
+            self.locationManager.requestWhenInUseAuthorization()
```

Comment:
should this also be requesting for "always allow" permissions, so beacons can be monitored in the background?

Reply:
This is probably a better question for @jrtibbetts. I don't know the exact requirements for beacons permissions on iOS. If it is like Android, then it does not necessarily need "background location."

Reply:
I think you're right, @jsani-radar. [Apple's docs](https://developer.apple.com/documentation/corelocation/determining_the_proximity_to_an_ibeacon_device) state that:

> Important
Apps must have authorization to use region monitoring, and **they must be configured with the Location updates background mode** to be launched. For more information, see [Requesting Authorization for Location Services](https://developer.apple.com/documentation/corelocation/requesting_authorization_for_location_services).

(emphasis mine)